letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -296,11 +314,24 @@ elif [ -f /etc/mageia-release ]; then
   }
   BOOTSTRAP_VERSION="BootstrapMageiaCommon $BOOTSTRAP_MAGEIA_COMMON_VERSION"
 elif [ -f /etc/redhat-release ]; then
-  Bootstrap() {
-    BootstrapMessage "RedHat-based OSes"
-    BootstrapRpmCommon
-  }
-  BOOTSTRAP_VERSION="BootstrapRpmCommon $BOOTSTRAP_RPM_COMMON_VERSION"
+  PREV_LE_PYTHON="$LE_PYTHON"
+  DeterminePython2Version "NOCRASH"
+  if [ "$PYVER" -eq 26 ] || [ "$PYVER" -eq 0 ]; then
```

Comment:
Did you find Red Hat systems that might not have Python installed? Is our Python 3 approach expected to work on them?

Reply:
We didn't previously assume that python would be installed, given that we try to install python as a dependency, so I was trying to keep that property. Unless we did assume that implicitly somewhere that I'd missed?

Reply:
We didn't previously assume that, however, we do assume that [yum](https://en.wikipedia.org/wiki/Yellow_Dog_Updater,_Modified) is installed which (at least nowadays) is written in Python so it may be safe to make that assumption.

I'm just not sure if the `"$PYVER" -eq 0` path will ever be used and if it is, if it's even the right thing to do. Do we expect a red hat system without a Python to be CentOS/RHEL 6 where we can install python34 from EPEL?

Reply:
Yes, that is exactly what I was imagining. I didn't know that yum has python as a dependency.

Reply:
I'm happy to remove the check. Worst thing that happens is that we accidentally fall through, install python2.6, then next time around we rebootstrap.

Reply:
Actually, if I change this to `if [ "$PYVER" -le 26 ]` and fix up the other function, we could theoretically just as easily support people who have no python, python 2.6, or like some weird python 2.5 or whatever if that's even a thing, all just as easily, by installing python3 for them. Why not do that?

Reply:
If there's a way we can easily do that, I'm certainly not against it, however, I'm not sure it's that simple because we're installing Python 3.4 using Red Hat tooling, package names, and a repository that only works on systems based on RHEL 6.

Reply:
yeah ok supporting more things somehow sounds less fun, I'm ripping this nocrash stuff out.